dataland-frontend/src/components/forms/parts/fields/YesNoNaBaseDataPointFormField.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="mb-3" :data-test="name">
+    <BaseDataPointFormField
+      :name="name"
+      :description="description"
+      :label="label"
+      :required="required"
+      @reports-updated="reportsUpdated"
+    >
+      <YesNoNaFormField
+        name="value"
+        :label="label"
+        :description="description"
+        :is-required="required"
+        :validation="validation"
+        :validation-label="validationLabel ?? label"
+        classes=""
+      />
+    </BaseDataPointFormField>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { BaseFormFieldProps } from "@/components/forms/parts/fields/FormFieldProps";
+import BaseDataPointFormField from "@/components/forms/parts/elements/basic/BaseDataPointFormField.vue";
+import { type DocumentToUpload } from "@/utils/FileUploadUtils";
+import YesNoNaFormField from "@/components/forms/parts/fields/YesNoNaFormField.vue";
+
+export default defineComponent({
+  name: "YesNoNaBaseDataPointFormField",
+  components: { YesNoNaFormField, BaseDataPointFormField },
+  inheritAttrs: false,
+  props: { ...BaseFormFieldProps },
+  emits: ["reportsUpdated"],
+  methods: {
+    /**
+     * Emits event that the selected document changed
```

Comment:
This function is duplicated in another component

Reply:
We could just drop it and emit emidiately on call.